docker/docker-compose.yml

```diff
@@ -263,3 +265,47 @@ services:
       - ${MF_COAP_ADAPTER_PORT}:${MF_COAP_ADAPTER_PORT}/tcp
     networks:
       - mainflux-base-net
+
+  twins-db:
+    image: mongo:3.6-jessie
```

Comment:
Although I have nothing against, I would advise using latest stable - 4.2: https://docs.mongodb.com/manual/release-notes/

Also, it looks like for some reason they prefer Ubuntu (Bionic) then Debian Jessie by default: https://hub.docker.com/_/mongo

Reply:
Will be fixed.